package.json

```diff
@@ -0,0 +1,75 @@
+{
+  "name": "mozilla-idea-town-app",
+  "description": "dom twiddling for the town of ideas",
+  "version": "1.0.0",
+  "author": "Mozilla (https://mozilla.org/)",
+  "bugs": {
+    "url": "https://github.com/mozilla/idea-town-app/issues"
+  },
+  "dependencies": {
+    "ampersand": "^3.0.5",
+    "ampersand-app": "^1.0.4",
+    "ampersand-collection": "^1.4.5",
+    "ampersand-collection-view": "^1.4.0",
+    "ampersand-dom": "^1.4.0",
+    "ampersand-events": "^1.1.1",
+    "ampersand-model": "^5.0.3",
+    "ampersand-router": "^3.0.2",
+    "ampersand-state": "^4.5.6",
+    "ampersand-view": "^8.0.1",
+    "ampersand-view-switcher": "^2.0.0",
+    "babel-core": "^5.8.22",
+    "bluebird": "^2.9.34",
+    "browserify": "^11.0.1",
+    "domready": "^1.0.8",
+    "globby": "^2.1.0",
+    "gulp-if": "^1.2.5",
+    "mozilla-tabzilla": "^0.4.0",
+    "mustache": "^2.1.3",
+    "node-bourbon": "^4.2.3",
+    "node-neat": "^1.7.2",
+    "node-sass": "^3.2.0",
+    "normalize.css": "^3.0.3",
+    "through2": "^2.0.0"
+  },
+  "devDependencies": {
+    "babel": "^5.8.21",
+    "babel-eslint": "4.0.10",
+    "babelify": "^6.1.3",
+    "browserify": "^11.0.1",
```

Comment:
@pdehaan Thanks for pointing that out - I've seen it and ignored it for a while now ;-)

Question: do we really need `devDependencies` at all? We aren't planning on publishing this project as a module, so I don't see the need for it. Quoth [the docs](https://docs.npmjs.com/files/package.json#devdependencies),

> If someone is planning on downloading and using your module in their program, then they probably don't want or need to download and build the external test or documentation framework that you use.

That doesn't apply here at all. Our only consumers will be ops, who will probably just point a DNS name at an S3 bucket (or CloudFront / S3 setup) containing the built files.

I guess there's some vague stylistic niceness about keeping `gulp-*` in devDependencies, makes package.json easier to parse?

Or maybe I'll just get rid of devDependencies for this repo. Thoughts?